src/os/alert/alertmanagerinstance.js

```diff
@@ -0,0 +1,12 @@
+goog.module('os.alertManager');
+goog.module.declareLegacyNamespace();
+
+const AlertManager = goog.require('os.alert.AlertManager');
+
+/**
+ * Global alert manager reference.
+ * @type {AlertManager}
+ */
+const instance = AlertManager.getInstance();
+
+exports = instance;
```

Comment:
Is there no way to have multiple different exports from the root class file? Having to add an instance creator like this seems superfluous.

Reply:
I sort of explained this in https://github.com/ngageoint/opensphere/pull/807#discussion_r352179076, but basically I didn't want breaking changes in this PR. There is a _much_ better way to define singletons, but since we have code doing `goog.require('os.alertManager')` I wanted to preserve that namespace and you can't declare multiple modules/namespaces per file.

You can export as many things as you want, for example this is perfect valid:
```
exports.SOME_CONSTANT = 42;
exports.SomeClass = class MyClass { ... };
exports.someFunction = function() { ... };
```

Reply:
I made a change to replace `goog.addSingletonGetter` with a `static getInstance()` function on the class. The second instance file is still there for backward compatibility, with a note up top that we should drop it. I can do that refactor later, but I wanted to limit changes in this PR to converted files where possible.